### ISSUE TITLE
Frames folder not available

### ISSUE THREAD

**Issue by jeroenburgt** [mhug-Trento/DL4covidUltrasound#3]
When logging into the ICLUS database( https://iclus-web.bluetensor.ai/login/) the frames folder is not present as described in point 6.

6. Usage
Frame-based Score Prediction
To use the exact train-test split used in our paper, please download the frames folder containing the extracted frames from this link and place the folder under the dataset/ folder.

The frame-based score predictor can be trained by running the following commands from the root folder:

**Comment by ResonWang** [comment]
yes, I have the same problem. From the dataset.csv It can be found that the mentioned folder contained the all data used in the paper(58000+) and by now it seems that the authors have decided to only open part of them(60 videos). So I think if we want to use the dataset, we have to exact the frames from the videos and label them by ourselves.

**Comment by roysubhankar** [comment]
@jeroenburgt @TonyApril 
Hi, apologies for the inconvenience. The dataset/ folder hasn't been uploaded yet. Could you please write an email to libertario.demi@unitn.it to inquire about the dataset, since he is the person responsible for uploading the dataset to the iclus website.

**Comment by kachiO** [comment]
Yeah I doubt they will release the dataset. I’ve been waiting for months now 😔 (https://github.com/mhug-Trento/DL4covidUltrasound/issues/1#issue-62119684)

**Comment by samueld62123** [comment]
They claimed in the paper about releasing the dataset, however they did not. Seems like they are waiting for the end of covid19.

**Comment by maria-zafar** [comment]
> yes, I have the same problem. From the dataset.csv It can be found that the mentioned folder contained the all data used in the paper(58000+) and by now it seems that the authors have decided to only open part of them(60 videos). So I think if we want to use the dataset, we have to exact the frames from the videos and label them by ourselves.
Is anybody able to get frames yet?

**Comment by maria-zafar** [comment]
Is anybody able to get frames yet?
